manager/api/micro/pom.xml

```diff
@@ -238,5 +239,31 @@
         <filtering>true</filtering>
       </resource>
     </resources>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <instructions>
+          <Import-Package>
+            javax.servlet;version="[2.6,4)",
+            javax.servlet.http;version="[2.6,4)",
+            org.eclipse.jetty.*;version="[8.0,10)",
```

Comment:
I think the ranges specified (for these packages) is good enough.  It's only going to be a problem for major version upgrades of core technologies.  So e.g. when we want to move to servlet 5 or something.  The only one that might be an issue is swagger, since the upper bound is 2.0 and that's currently the version we need.

To answer the question - I don't know. :)